static/src/javascripts/projects/common/modules/commercial/third-party-tags.js

```diff
@@ -28,13 +33,28 @@ define([
     krux,
     identity,
     outbrain,
-    plista) {
+    plista,
+    externalContentContainerStr,
+    fastdom,
+    template) {
 
     function loadExternalContentWidget() {
+
+        var externalTpl = template(externalContentContainerStr);
+        var documentAnchorId = '.js-external-content-widget-anchor';
+
+        function renderWidgetContainer(widgetType) {
+            $(documentAnchorId).append(externalTpl({widgetType: widgetType}));
+        }
+
         if (config.switches.plistaForOutbrainAu && config.page.edition.toLowerCase() === 'au') {
-            plista.init();
+            fastdom.write(function () {
+                renderWidgetContainer('plista');
+            }).then(plista.init());
```

Comment:
You might want  `then(plista.init)` here, otherwise it'll be invoked on parse rather than after `then`.